### PR TITLE
Add llama3-8b vLLM support

### PR DESCRIPTION
**Pull request by jjk-g**
Fixes https://github.com/kubernetes-sigs/wg-serving/issues/4

### CODE REVIEW

File: serving-catalog/core/deployment/vllm/llama3-8b/gke/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+resources:
+  - ../base
+
+components:
+  - ../../../components/gke/resources/gpu/1-L4
+
+patches:
+  - path: deployment.patch.yaml
+    target:
+      kind: Deployment
```
Comment (ahg-g) [inline serving-catalog/core/deployment/vllm/llama3-8b/gke/kustomization.yaml:14]: remove?

**Comment by jjk-g** [comment]
@ahg-g Is this still meant to be on hold?

**Comment by ahg-g** [comment]
no

/hold cancel

**Pull request merged**